app/views/user_mailer/send_email.html.erb

```diff
@@ -6,5 +6,6 @@
   <body>
     <h1>Sorry to inform you, <%= @recipient.name %></h1>
     <p><%= @user.name %> has died :( </p>
+    <p> Here is a link to <%= @user.name %>'s   <a href =<%= @user_url %>> Memorial Page </a> </p>
```

Comment:
Just curious, is it possible to add styling to this html page? Not a priority or anything but wondering if thats possible

Reply:
We should actually make a plain text version of this, a fair amount of email clients won't accept the html version. Basically you just copy paste all the erb stuff but remove any html elements ``` <h1>, <p>, etc. ```